webextensions/api/runtime.json

```diff
@@ -50,6 +88,90 @@
               }
             }
           },
+          "id": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "26"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "45"
+                },
+                "firefox_android": {
+                  "version_added": "48"
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "14"
+                },
+                "safari_ios": {
+                  "version_added": "15"
+                }
+              }
+            }
+          },
+          "nativeApplication": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "74"
```

Comment:
Despite "nativeApplication" being part of Chrome's API docs, this is not a feature that third party extension developers can use.

It is only implemented on ChromeOS, and  not documented further. Last time I checked, only a few internal extensions can use this feature.

Reply:
This feature is not documented and hasn't even shipped to stable Chrome versions: https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/extensions/api/_manifest_features.json;l=170-173;drc=63c3a9bcd07af895d8f40b75111b131f742a89f3

Let's remove the `nativeApplication` entry altogether.

Reply:
However, in the Chrome docs:
![image](https://github.com/mdn/browser-compat-data/assets/7352080/991b4078-02a8-455c-a94f-0b4fefcedc46)

Reply:
I've already notified the Chrome extensions team of their documentation issue.

Here is another independent experience from a third-party (extension developer) that confirms the restricted API: https://github.com/w3c/webextensions/issues/457#issuecomment-1735974873

Some more details:

The `MessageSender.nativeApplication` fieldis only set in `port` instances passed to the `runtime.onConnectNative` event, an event that is fired when the native application opened the connection to the extension. This feature requires presence of the `natively_connectable` manifest key (with a listing of permitted native messaging application IDs), which is currently restricted to Chrome's dev channel (i.e. unavailable on release).